src/httpserver.cpp

```diff
@@ -331,6 +337,8 @@ static bool HTTPBindAddresses(struct evhttp* http)
     } else if (mapMultiArgs.count("-rpcbind")) { // Specific bind address
         const std::vector<std::string>& vbind = mapMultiArgs.at("-rpcbind");
         for (std::vector<std::string>::const_iterator i = vbind.begin(); i != vbind.end(); ++i) {
+            if (boost::starts_with(*i, RPC_ADDR_PREFIX_UNIX)) // Skip UNIX sockets here
```

Comment:
I like boost::starts_with. It's python idom (and many other languages) If we want to do without that, I think we should simply implement our own in terms of what you write.